src/client/client4.js

```diff
@@ -2661,6 +2661,15 @@ export default class Client4 {
         );
     }
 
+    installMarketplacePlugin = async (id, version) => {
+        this.trackEvent('api', 'api_install_marketplace_plugin');
```

Comment:
I believe we are already tracking this on the webapp [here](https://github.com/mattermost/mattermost-webapp/blob/23648a1d3cb8c3f31adcc70a37056111afcc9f24/components/plugin_marketplace/marketplace_item/marketplace_item.jsx#L88)

Reply:
true. However if someone uses this directly by passing the webapp we'll be missing this event. But don't mind removing it :P

Reply:
Agreed. We should remove the one in the webapp then.

Reply:
@marianunez 
I'm personally OK keeping both, because once 5.16 is released, our analytics tool will contain this event. However >= 5.18 it'll be the other redux event, it might be hard to put together that data if you need to see how many people installed a marketplace plugin. My suggestion would to keep both now. If this change was going into 5.16 then it would've been fine removing that event.

Reply:
There still needs to be a change to add the event to the whitelist [here](https://github.com/mattermost/mattermost-redux/blob/e40072b407f8d437cb10a2edc78ac7972c7587be/src/client/client4.js#L2946) so that it can actually get logged - given that most events right now are bypassed to reduce usage in segment. 

@aaronrothschild could you confirm if we should include this event to the whitelist to get tracked?

Reply:
Ah i see. In that case I shall remove.

Reply:
@ali-farooq0 Which one are you referring that will be removed?

Reply:
@marianunez the webapp one

Reply:
Gotcha, well the whitelist would need to be updated to include your new API event `api_install_marketplace_plugin` instead of [this one](https://github.com/mattermost/mattermost-redux/blob/e40072b407f8d437cb10a2edc78ac7972c7587be/src/client/client4.js#L2942)

Reply:
ah. forgot about that as well. Good catch! Maybe i should revert my other webapp change to remove the tracking there? or leave it the way it is right now?

Reply:
0/5 either way. We just need to make sure that whichever we keep we add to the whitelist so that it actually gets to segment:

`ui_marketplace_download` and/or
`api_install_marketplace_plugin`

@aaronrothschild thoughts?

Reply:
@marianunez shall we unblock this PR and I can update the tracking events once I get the confirmation from Aaron.

I've created a jira to keep track and I can follow up with Aaron later, does that work?
https://mattermost.atlassian.net/browse/MM-19318

Reply:
Sure, we just would need to add the new event in the whitelist then because now with `ui_marketplace_download` we are not tracking either at this point🤔

Reply:
ah i didn't realize that ui event was whitelisted :(. I've reverted my change in webapp. https://github.com/mattermost/mattermost-webapp/pull/3871/commits/0a5ec12ca7b84b080c8eb42953a94ccd5e4e3c27